### PR TITLE
Fix ARTIK download script.

### PR DESCRIPTION
**Pull request by btheosam**
ARTIK download script uses hardcoded partition name for ALL options.
Since the partition names are part of Kconfig file, it is NOT correct
to use hard coded values. This commit reads the partition names
from .config file if present, otherwise it reads the default parttion
names from Kconfig file.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [be7303b6252bc0d70e97608e62ed38f39c390ba3] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [be7303b6252bc0d70e97608e62ed38f39c390ba3] - Code Rule Check (C++) OK.

File: build/configs/artik05x/artik05x_download.sh [hunk 1]
```diff
@@ -173,11 +129,11 @@ compute_ocd_commands()
 
 download()
 {
-	# Make a list which part should be flashed
-	parts=$(compute_fw_parts $1)
-	echo "The \"${parts}\" partition(s) will be flashed"
+    parts_default=`grep -A 2 'config ARTIK05X_FLASH_PART_NAME' ${BOARD_KCONFIG} | sed -n 's/\tdefault "\(.*\)".*/\1/p'`
```
Comment (jsdosa) [inline build/configs/artik05x/artik05x_download.sh]: could you check for indent~?

**Comment by seinfra** [comment]
Target : [fe364d5d1f8e1c27b85822daf91ae7c151428e81] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [fe364d5d1f8e1c27b85822daf91ae7c151428e81] - Code Rule Check (C++) OK.

**Pull request merged**